lib/src/abi_specific_ints.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:ffi';
+
+// `char` in C.
+typedef Char = Int8;
```

Comment:
The signedness of `char` type in `C` is arch/platform specific. Do all of our arch/plat combinations specify it as signed?

Reply:
You're right. Fixed in the SDK CL.